man/oracle.md

```diff
@@ -10,99 +10,56 @@ oracle - server for the SPHINX password manager
 
 # DESCRIPTION
 
-The SPHINX protocol only makes sense if the server (called oracle) is
-somewhere else than where you type your password, pwdsphinx comes with
-a server implemented in python3 which you can host off-site from your
-usual desktop/smartphone.
+The SPHINX protocol only makes sense if the server (called *oracle*) is located somewhere other than where you type your password. `pwdsphinx` comes with a server implemented in Python 3, which you can host off-site from your usual desktop or smartphone.
```

Comment:
i guess it should also mention the zphinx zerver which is better suited for production systems...